x-pack/packetbeat/cmd/root.go

```diff
@@ -20,7 +27,20 @@ var Name = packetbeatCmd.Name
 // RootCmd to handle beats cli
 var RootCmd *cmd.BeatsRootCmd
 
+// packetbeatCfg is a callback registered via SetTransform that returns a packetbeat Elastic Agent client.Unit
+// configuration generated from a raw Elastic Agent config
+func packetbeatCfg(rawIn *proto.UnitExpectedConfig, _ *client.AgentInfo) ([]*reload.ConfigWithMeta, error) {
```

Comment:
Caller before callee makes context easier to grasp, though in this situation there is ~0 difference.